products/cloudfunctions/api.yaml

```diff
@@ -0,0 +1,156 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: CloudFunctions
+display_name: Cloud Functions
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://cloudfunctions.googleapis.com/v1/
+scopes:
+  - https://www.googleapis.com/auth/cloudfunctions
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Cloud Pub/Sub API
```

Comment:
Why is this called Pub/Sub? API URL looks like a CloudFunctions link.

Reply:
Copy and paste error! Good catch